### DISCUSSION TITLE
Snare - Can't connect to tanner host

### DISCUSSION THREAD

**Discussion by thinkpadre** [telekom-security/tpotce#805] [Q&A]
Hi, 
I have issue with my Tpot
I want to add custom(example.com) website to snare. I modify DockerFile for snare 

I remove # 
`clone --target example.com && \  `

And add --page-dir example.com
`
CMD snare --tanner tanner --debug true --no-dorks true --auto-update false --host-ip 0.0.0.0 --port 80 --page-dir example.com
`

i used command docker-compose build snare and i also tried command docker-compose build

all time i have this error
![image](https://user-images.githubusercontent.com/71973199/112876866-50e45e00-90c6-11eb-8a59-d666039c0eab.png)

**Comment by t3chn0m4g3** [comment]
This is not an issue with T-Pot, adjustments have been discussed before, please search the issues or discussions. Also see the Wiki for docker volumes examples and the Snare / Tanner repos on GitHub for their docs.
